gulpfile.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+const fs = require('fs');
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+
+gulp.task('test', () =>
+  gulp.src('test/*_test.js', {read: false})
+      .pipe(mocha({
+        ui: 'tdd',
```

Comment:
yeah... that's how I've been doing it, but lately I've been leaning towards requiring fewer global installs. Just gulp, then everything goes from there. I don't feel too strongly though.